src/components/aeAmountInput/aeAmountInput.md

```diff
@@ -1,18 +1,19 @@
-```javascript
-new Vue({
-  data() {
-    return {
-      amount: 1.337,
-      step: 0.01
-    }
-  },
-  template: `
-    <ae-amount-input :value="amount" :step="step" :symbol="'ETH'" @input='changeInput' />
-  `,
-    methods: {
-        changeInput: function(newAmount) {
-            this.amount = newAmount
-        }
-    }
-})
+```js
+  new Vue({
+    data: () => ({ value: undefined }),
+    template: `
+      <div>
+        <ae-amount-input
+          placeholder="0.00"
+          v-model="value"
+          :tokens="[
```

Comment:
I think we should change the name to something more abstract. What if i want to use it to input lengths, 
```
:tokens="[{ symbol: 'CM', name: 'centimeters' },{ symbol: 'IN', name: 'inches' }]
```
maybe we better call it "unit"?

Reply:
see #50